module/Decision/src/Mapper/Member.php

```diff
@@ -50,10 +50,16 @@ public function searchByName(
             ->addScalarResult('generation', 'generation', 'integer');
 
         $sql = <<<QUERY
-            SELECT `lidnr`, CONCAT_WS(' ', `firstName`, IF(LENGTH(`middleName`), `middleName`, NULL), `lastName`) as `fullName`, `generation`
+            SELECT `lidnr`,
+                CONCAT_WS(' ', `firstName`, IF(LENGTH(`middleName`), `middleName`, NULL), `lastName`) as `fullName`,
+                `generation`
             FROM `Member`
-            WHERE CONCAT(LOWER(`firstName`), ' ', LOWER(`lastName`)) LIKE :name
-            OR CONCAT(LOWER(`firstName`), ' ', LOWER(`middleName`), ' ', LOWER(`lastName`)) LIKE :name
+            WHERE
+                (
+                CONCAT(LOWER(`firstName`), ' ', LOWER(`lastName`)) LIKE :name
+                OR CONCAT(LOWER(`firstName`), ' ', LOWER(`middleName`), ' ', LOWER(`lastName`)) LIKE :name
```

Comment:
I don't think there is a specific SQL statement formatting standard, but I settled for parentheses-are-the-only-character-on-a-line and indent-on-every-keyword